config.json

```diff
@@ -902,7 +902,11 @@
       "core": false,
       "difficulty": 8,
       "slug": "rectangles",
-      "topics": null,
+      "topics": [
+        "algorithms",
+        "logic",
+        "pattern_recognition"
```

Comment:
Good suggestions :) I agree with `logic` and `pattern_recognition`. I would also add `strings` and `arrays`. I'm not sure about `algorithms`. You could technically add that as a topic to every exercise but I'm not sure that would be very useful. So I think it's best to only add `algorithms` to exercises where you have to use a specific algorithm which I don't think is true for this exercise.

Reply:
I understand what you mean. I will update once I am home. 

In the meanwhile, I suppose it would be good to update the other projects which have the same tags, don't you think? [The python config](https://github.com/exercism/python/blob/master/config.json#L729) has the topics that I have added in this PR

Reply:
No, the topics should not be the same across tracks. The topics are meant to be topics you might encounter when solving the exercise in this specific language. Those topics might very well be relevant to the python exercise. Therefore, to choose topics you should look at the Java implementation of the tests and the reference implementation (as explained in the issue). Does that make sense? :)

Reply:
I understand. Thank you for explaining, I will update it accordingly!